### ISSUE TITLE
Your board support is limited

### ISSUE THREAD

**Issue by Tofally** [Rem0o/FanControl.Releases#1001]
Hello. 

Sorry to bother you again but I can't seem to get things working. I did install a new hardware monitor and replaced the file with the new one in Fan Control files. What do you exactly mean by: "Your board support is limited"? I really would like to start using Fan Control but I just can't and I would really thank you for your help. 

My first issue report: https://github.com/Rem0o/FanControl.Releases/issues/992
My second issue report: https://github.com/Rem0o/FanControl.Releases/issues/998

Tofally

**Comment by jaivu** [comment]
Try installing cpu-z and then fan control.

**Comment by Tofally** [comment]
As always, it did not work. But still, thanks for the help. When I tried one of my fans in my brother's PC it did work. We have the same brand and about the same year's motherboards. The bios were identical. Fan Control worked on his pc when I turned fans from bios to 50%. I am so close to just formatting my  whole pc but I guess I will just not let myself do that and keep using Speedfan if I can't get it to work. And if you ask then yes I have deleted Speedfan, restarted my PC, and downloaded Fan Control so Speedfan should not be the issue here.
